[2025-01-07T14:53:00.345+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-01-07T14:53:00.444+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.run_my_function scheduled__2024-04-29T00:00:00+00:00 [queued]>
[2025-01-07T14:53:00.481+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.run_my_function scheduled__2024-04-29T00:00:00+00:00 [queued]>
[2025-01-07T14:53:00.482+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-01-07T14:53:00.563+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): run_my_function> on 2024-04-29 00:00:00+00:00
[2025-01-07T14:53:00.581+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'run_my_function', 'scheduled__2024-04-29T00:00:00+00:00', '--job-id', '224', '--raw', '--subdir', 'DAGS_FOLDER/dag_test.py', '--cfg-path', '/tmp/tmptsbc87qi']
[2025-01-07T14:53:00.581+0000] {standard_task_runner.py:63} INFO - Started process 727 to run task
[2025-01-07T14:53:00.586+0000] {standard_task_runner.py:91} INFO - Job 224: Subtask run_my_function
[2025-01-07T14:53:00.699+0000] {task_command.py:426} INFO - Running <TaskInstance: my_first_dag.run_my_function scheduled__2024-04-29T00:00:00+00:00 [running]> on host 4ff5b0867c78
[2025-01-07T14:53:01.049+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='run_my_function' AIRFLOW_CTX_EXECUTION_DATE='2024-04-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-29T00:00:00+00:00'
[2025-01-07T14:53:01.050+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-01-07T14:53:01.066+0000] {logging_mixin.py:188} INFO - Hello, this is my first DAG!
[2025-01-07T14:53:01.067+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-01-07T14:53:01.068+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-01-07T14:53:01.080+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=run_my_function, run_id=scheduled__2024-04-29T00:00:00+00:00, execution_date=20240429T000000, start_date=20250107T145300, end_date=20250107T145301
[2025-01-07T14:53:01.128+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-01-07T14:53:01.152+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-07T14:53:01.160+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
